.github/workflows/python-test.yml

```diff
@@ -92,26 +92,26 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: [3.8, 3.9]
+        python-version: ['3.9', '3.10', '3.11']
         include:
         - os: ubuntu-latest
           path: ~/.cache/pip
         - os: macos-latest
           path: ~/Library/Caches/pip
         - os: windows-latest
           path: ~\AppData\Local\pip\Cache
-        # optional 3.7 test
+        # optional
         - os: ubuntu-latest
           path: ~/.cache/pip
-          python-version: 3.7
-        # optional 3.7 test
+          python-version: '3.10'
+        # optional
         - os: macos-latest
           path: ~/Library/Caches/pip
-          python-version: 3.7.16
-        # optional 3.7 test
+          python-version: '3.10'
+        # optional
         - os: windows-latest
           path: ~\AppData\Local\pip\Cache
-          python-version: 3.7
```

Comment:
Same comment applies as to the other file that this block can be removed.

Reply:
Fixed.